.babelrc

```diff
@@ -1 +1,12 @@
-{ "presets": ["env"] }
+{ 
+  "presets": ["env"], 
+  "sourceMaps": true,
+    "retainLines": true,
+    "env": {
+        "test": {
+          "plugins": [
+            "istanbul"
```

Comment:
Does this plugin need to be installed first?

Reply:
Yeah! I have fixed it.